### ISSUE TITLE
Threading/tree sort

### ISSUE THREAD

**Issue by 4trdg** [Fiddlekins/SpookyX#9]
Do you plan to implement 4chan X's 'threading' function? I never got used to inline.

**Comment by Fiddlekins** [comment]
Can you take a screenshot of what you mean? I'm either not aware of the feature or have already implemented it (albeit buggily)

**Comment by 4trdg** [comment]
Here's a screenshot from 4chan x. Posts that are replies are indented automatically. https://imgur.com/a/mSiU5

**Comment by Fiddlekins** [comment]
Your image seems a little dead to me

**Comment by 4trdg** [comment]
Sorry, I edited the link.

**Comment by Fiddlekins** [comment]
Ah cool, I see now. SpookyX doesn't do that currently. How does the feature behave when a post replies to more than one post?

**Comment by 4trdg** [comment]
If the quotes are also replying to each other themselves then it nests the post under the replies. 
https://i.imgur.com/vQLmJou.png

Otherwise, if the quotes are in separate reply chains, it defaults to top-level.

**Comment by Fiddlekins** [comment]
Right, so rather than duplicating the post it'll put it in what it deems the most suitable place. I guess I'll take a look into adding this feature when I next get some spare time then.
